locale/ar/docs/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: التوثيق
+layout: docs.hbs
+labels:
+  lts: LTS
+---
+
+# عن التوثيق
+
+تتوفر عدة توثيقات على هذا الموقع:
+* التوثيق الخاص بمرجع واجهة برمجة التطبيق
+* مميزات ES6
+* الدلائل
+
+### التوثيق الخاص بمرجع واجهة برمجة التطبيق
+### API Reference Documentation
```

Comment:
L16 already translated in [L15](https://github.com/nodejs/nodejs.org/pull/2143/files#diff-aa61ba60f5910ed95d84926d23228c8fR15) !

Reply:
requested changes has been done to the file 💯